cargo-make/main.toml

```diff
@@ -0,0 +1,28 @@
+[env]
+RUSTFLAGS = ""
+LEPTOS_OUTPUT_NAME = "ci" # allows examples to check/build without cargo-leptos
+
+[env.github-actions]
+RUSTFLAGS = "-D warnings"
+
+[tasks.check]
+alias = "check-all"
+
+[tasks.check-all]
+command = "cargo"
+args = ["+nightly", "check-all-features"]
+install_crate = "cargo-all-features"
+
+# FIXME: Should this duplicate of check-all be removed?
```

Comment:
`check-all-release` was *supposed* to be the equivalent of `cargo check --release`, as opposed to `check-all` running the debug check. The release flag must have dropped out inadvertently at some point.

Reply:
I pushed up the change.